### PR TITLE
Tuxtest support for more android devices

### PR DESCRIPTION
**Pull request by stylesen**

### CODE REVIEW

File: lava_test_plans/projects/lkft-android/devices/variables/hi6220-hikey-r2.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+DEVICE_TYPE: hi6220-hikey
+KERNEL_URL: unknown
+DTB_URL: unknown
+BOOT_URL: unknown
+MODULES_URL: unknown
+ROOTFS_URL: unknown
+LAVA_JOB_PRIORITY: 25
+PROJECT: projects/lkft/
```
Comment (roxell) [inline lava_test_plans/projects/lkft-android/devices/variables/hi6220-hikey-r2.yaml:8]: should this be 'projects/lkft-android/' ?

File: lava_test_plans/projects/lkft-android/devices/variables/hi6220-hikey-r2.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+DEVICE_TYPE: hi6220-hikey
```
Comment (roxell) [inline lava_test_plans/projects/lkft-android/devices/variables/hi6220-hikey-r2.yaml:1]: should probably be 'hi6220-hikey-r2' ?

File: lava_test_plans/projects/lkft/devices/variables/hi6220-hikey-r2.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+DEVICE_TYPE: hi6220-hikey
```
Comment (roxell) [inline lava_test_plans/projects/lkft/devices/variables/hi6220-hikey-r2.yaml:1]: should probably be 'hi6220-hikey-r2' ?

Comment (stylesen) [reply lava_test_plans/projects/lkft/devices/variables/hi6220-hikey-r2.yaml:1]: done.

File: lava_test_plans/projects/lkft-android/devices/variables/hi6220-hikey-r2.yaml (hunk 2 above)
Comment (stylesen) [reply lava_test_plans/projects/lkft-android/devices/variables/hi6220-hikey-r2.yaml]: done.

File: lava_test_plans/projects/lkft-android/devices/variables/hi6220-hikey-r2.yaml (hunk 1 above)
Comment (stylesen) [reply lava_test_plans/projects/lkft-android/devices/variables/hi6220-hikey-r2.yaml]: done.

File: lava_test_plans/projects/lkft-android/devices/variables/hi6220-hikey-r2.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+DEVICE_TYPE: hi6220-hikey-r2
+KERNEL_URL: unknown
+DTB_URL: unknown
+BOOT_URL: unknown
+MODULES_URL: unknown
+ROOTFS_URL: unknown
```
Comment (roxell) [inline lava_test_plans/projects/lkft-android/devices/variables/hi6220-hikey-r2.yaml:6]: These variables should probably reflect what is in qrb5165-rb5 no?

**Pull request merged**